test/jdk/java/awt/Frame/MultiScreenTest.java

```diff
@@ -99,7 +99,7 @@ public List<JFrame> init() {
         for (int j = 0; j < gs.length; j++) {
             GraphicsConfiguration[] gc = gs[j].getConfigurations();
             if (gc.length > 0) {
-                for (int i = 0; i < gc.length / 2; i++) {
+                for (int i = 0; i < gc.length && i < 10; i++) {
                     JFrame f = new JFrame(gc[i]); //test JFrame( gc )
                     GCCanvas c = new GCCanvas(gc[i]);//test canvas( gc )
```

Comment:
For consistency, add a space before start of comment.
```suggestion
                    GCCanvas c = new GCCanvas(gc[i]); // test canvas( gc )
```